sys/build.rs

```diff
@@ -192,6 +228,12 @@ fn build_library(src_dir: &Path, lib_dir: &Path) {
         .files(extra_srcs)
         .out_dir(lib_dir);
 
+    #[cfg(not(target_os = "windows"))]
+    build
+        .flag("-std=c++11")
+        .flag("-ffunction-sections")
+        .flag("-fdata-sections");
+
```

Comment:
Seems needs skip that flags for `*-windows-msvc` target only because `*-windows-gnu` support it.
May be add equivalent flags also for `*-windows-msvc`.

Reply:
This complex logic might be best expressed through dynamic `contains()` tests on the `TARGET` env var.